### ISSUE TITLE
[AIR SDK 50.2.5.1][Linux] `Worker::start()` cause crash

### ISSUE THREAD

**Issue by itlancer** [airsdk/Adobe-Runtime-Support#3195]
## Problem Description
`Worker::start()` cause application crash using AIR 50.2.5.1 with Linux.

Tested with latest AIR 50.2.5.1 with Ubuntu 22.04.1 LTS x86_64 devices with different applications.
There is no such issue with Windows.
Also there is no such issue using AIR 50.2.4.5 or 51.0.0.4.

Related issues:
https://github.com/airsdk/Adobe-Runtime-Support/issues/3166
https://github.com/airsdk/Adobe-Runtime-Support/issues/2523

## Steps to Reproduce
Launch application with code below on any Linux device.

Application example with sources and crash log attached.
[linux_worker_start_crash.zip](https://github.com/airsdk/Adobe-Runtime-Support/files/15065519/linux_worker_start_crash.zip)

```
package {
	import flash.display.Sprite;
	import flash.events.Event;
	import flash.system.Worker;
	import flash.system.WorkerDomain;
	
	public class LinuxWorkerStartCrash extends Sprite {
		
		public function LinuxWorkerStartCrash() {
			if (Worker.current.isPrimordial) {
				addEventListener(Event.ADDED_TO_STAGE, init);
			} else {
				trace("worker");
			}
		}
		
		private function init(e:Event):void {
			removeEventListener(Event.ADDED_TO_STAGE, init);
			
			var worker:Worker = WorkerDomain.current.createWorker(this.loaderInfo.bytes);
			worker.start();//This line cause crash
		}
	}
}
```

**Actual Result:**
Application crash
`Application crashed with an unhandled SIGSEGV`
Full crash log attached in zip archive above.

**Expected Result:**
Worker will be started without crash.

## Known Workarounds
none
*Use AIR 50.2.4.5

**Comment by ajwfrost** [comment]
This is a side-effect of the fix for #3126 ... will get a fix out for this in the next release.

**Comment by itlancer** [comment]
Fixed with latest AIR SDK 51.0.1.1.
Thanks!
